### ISSUE TITLE
Erin McMackin: Cover Letter

### ISSUE THREAD

**Issue by erinmcmackin** [ga-students/wdiroutcomes#138]
https://drive.google.com/open?id=12IQwZ6yAQKGpK4FMW9n3AgPNIsY6CI1T

http://www.tomesoftware.com/software-engineer-work-bicycle-projects/

*this is an old job posting, but I'm interested in some of the local companies in the next town over that often just ask for an email; using this one as a guide

**Comment by nicoleump** [comment]
Overall great job, Erin. 

For the second paragraph, you are immediately putting yourself into the newbie category. Perhaps reword it to talk about adding to your skillset by learning web development, including specific languages that are mentioned in the job ad.
